BuildScripts/BuildDfastmi.bat

```diff
@@ -17,6 +17,13 @@ START /B /WAIT poetry run nuitka ^
  --include-module=geopandas ^
  --include-package-data=geopandas.datasets ^
  --include-module=fiona ^
+ --company-name=Deltares ^
+ --file-version=3.0.0 ^
+ --product-version=2024.01.3.0 ^
```

Comment:
what is the difference between product version and file version?
Maybe something to discuss with Bert in the future?

Reply:
file version = the version of .exe
product version = the version of the actual product